### PR TITLE
Add Parrot Community image

### PR DESCRIPTION
**Pull request by kiazhi**
## PR Summary

- Add Parrot 4.2.2 image to `release/community-stable` for #67

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [x] Not Applicable
- **OR**
  - [ ] Update [README.powershell.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershell.md)
  - [x] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/community-stable/parrot/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      org.label-schema.vendor="PowerShell" \
+      org.label-schema.vcs-ref=${VCS_REF} \
+      org.label-schema.version=${PS_VERSION} \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.docker.cmd="docker run ${IMAGE_NAME} pwsh -c '$psversiontable'" \
+      org.label-schema.docker.cmd.devel="docker run ${IMAGE_NAME}" \
+      org.label-schema.docker.cmd.test="docker run ${IMAGE_NAME} pwsh -c Invoke-Pester" \
+      org.label-schema.docker.cmd.help="docker run ${IMAGE_NAME} pwsh -c Get-Help"
+
+CMD [ "pwsh" ]
```
Comment (TravisEz13) [inline release/community-stable/parrot/docker/Dockerfile]: missing an new line at the end of the file

**Comment by TravisEz13** [comment]
I'm going to wait to merge this until I have CI work for the community images.

**Comment by TravisEz13** [comment]
Can you add the equivalent of this change for this and the other new images you have (might be better to do one at a time as it might cause merge conflicts)?
https://github.com/TravisEz13/PowerShell-Docker/blob/32c0e7f50581c678590dc37cc8bb029fad0a33e3/vsts-ci.yml#L82-L89

**Comment by kiazhi** [comment]
I will do the Arch Linux image and followed by BlackArch Linux image.

There is an issue with the `bash` in the Parrot image where the `CMD [ "pwsh" ]` defined in Dockerfile is not honored and will immediately fail all those tests.

I'm still investigating why it will always use `bash` when accompanied with `docker run --rm [IMAGE:TAG] [COMMAND} {PARAMS}` .

It does not mean that `pwsh` is not working because it will still work as long as you run it interactively into the `bash` and type `pwsh`.

More work is needed to determine if I can fix this or trash this PR for the time being... I have marked this PR with WIP (Work In Progress) in title.

**Comment by kiazhi** [comment]
Fixed the issue regarding the parrot container not honoring `CMD [ "pwsh" ]` in the dockerfile.
- Add the `ENTRYPOINT []` after the `FROM` in the dockerfile to reset the entrypoint that is defined in the original image that was causing the issue.
https://github.com/ParrotSec/docker-images/blob/8cf2d26e8dd9b032dc6845ca8ab3076c4d07cf4b/parrot-core/Dockerfile#L26

All good now and ready to be merged.

**Comment by TravisEz13** [comment]
Thanks for all your contributions and working with me to get everything working.

**Pull request merged**